wagtail/admin/templates/wagtailadmin/pages/workflow_history/detail.html

```diff
@@ -1,21 +1,20 @@
 {% extends "wagtailadmin/base.html" %}
 {% load i18n wagtailadmin_tags %}
 
-{% block titletag %}{% blocktrans with title=page.get_admin_display_title %}Workflow progress {{ title }}{% endblocktrans %}{% endblock %}
+{% block titletag %}{% blocktrans %}Workflow progress{% endblocktrans %}{% endblock %}
```

Comment:
this would be better as `{% trans "Workflow progress" %}`

Reply:
Oh yes, thanks!